receiver/influxdbreceiver/receiver.go

```diff
@@ -165,7 +181,9 @@ func (r *metricsReceiver) handleWrite(w http.ResponseWriter, req *http.Request)
 		}
 	}
 
-	if err := r.nextConsumer.ConsumeMetrics(req.Context(), batch.GetMetrics()); err != nil {
+	err := r.nextConsumer.ConsumeMetrics(req.Context(), batch.GetMetrics())
+	r.obsrecv.EndMetricsOp(ctx, "influxdb", batch.GetMetrics().DataPointCount(), err)
```

Comment:
It might be a good idea to put `"influxdb"` in a string constant.

Reply:
Done.